gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/AppCdsSettings.kt

```diff
@@ -0,0 +1,181 @@
+package org.jetbrains.compose.desktop.application.dsl
+
+import org.gradle.api.file.Directory
+import org.gradle.api.file.RegularFile
+import org.jetbrains.compose.desktop.application.internal.JvmApplicationContext
+import org.jetbrains.compose.internal.utils.packagedAppJarFilesDir
+import java.io.Serializable
+
+/**
+ * The configuration of AppCDS for the native distribution.
```

Comment:
This needs to be described more here, why people need AppCDS - to increase the startup time of the application. Otherwise an average user just skip this property.

Reply:
Added a sentence describing what AppCDS is for.